email_handler.py

```diff
@@ -616,6 +665,11 @@ def handle_forward(envelope, msg: Message, rcpt_to: str) -> List[Tuple[bool, str
         # do not return 5** to allow user to receive emails later when alias is enabled or contact is unblocked
         return [(True, res_status)]
 
+    # Check if we need to reject or quarantine based on dmarc
```

Comment:
I think we can create a new config to turn on/off this dmarc anti-phishing check. We might need to collect the metrics about dmarc for several days before turning on this check.